modules/java/generator/src/cpp/listconverters.cpp

```diff
@@ -26,10 +42,23 @@ std::vector<float> List_to_vector_float(JNIEnv* env, jfloatArray arr)
     V.push_back(body[i]);
 
   env->ReleaseFloatArrayElements(arr, body, 0);
-  
   return V;
 }
 
+jlongArray vector_vpHomogeneousMatrix_to_List(JNIEnv* env, std::vector<vpHomogeneousMatrix> V)
```

Comment:
Is this correct? `jlongArray` should hold signed 64 bits integer. Is it not `jdoubleArray` instead?